core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -285,6 +392,16 @@ object ConfigCommand extends Config {
             .withRequiredArg
             .describedAs("urls")
             .ofType(classOf[String])
+    val bootstrapServerOpt = parser.accepts("bootstrap-server", "The Kafka server to connect to. " +
+      "This is required for describing and altering broker configs.")
+      .withRequiredArg
+      .describedAs("server to connect to")
+      .ofType(classOf[String])
+    val commandConfigOpt = parser.accepts("command-config", "Property file containing configs to be passed to Admin Client. " +
```

Comment:
Looks like this is the first case where we pass admin client properties to a tool, so I'm wondering about the name. An alternative might be `admin-config` to make it's relation to the AdminClient a bit more obvious.

Reply:
I had suggested `adminclient-config` for KIP-248 and later realized that `DelegationTokenCommand`, `ConsumerGroupCommand`, `BrokerApiVersionsCommand` and `DeleteRecordsCommand` use `--command-config`.

Reply:
Ok, guess the ship has sailed; "command-config" it is.